lib/ably/rest/client.rb

```diff
@@ -560,37 +553,37 @@ def raw_request(method, path, params = {}, options = {})
       # Sends HTTP request to connection end point
       # Connection failures will automatically be reattempted until thresholds are met
       def send_request(method, path, params, options)
-        max_retry_count    = http_defaults.fetch(:max_retry_count)
+        max_retry_count = http_defaults.fetch(:max_retry_count)
         max_retry_duration = http_defaults.fetch(:max_retry_duration)
-        requested_at       = Time.now
-        retry_count        = 0
-        retry_sequence_id  = nil
-        request_id         = SecureRandom.urlsafe_base64(10) if add_request_ids
+        requested_at = Time.now
+        retry_count = 0
+        retry_sequence_id = nil
+        request_id = SecureRandom.urlsafe_base64(10) if add_request_ids
 
         preferred_fallback_connection_for_first_request = get_preferred_fallback_connection_object
 
         begin
-          use_fallback = can_fallback_to_alternate_ably_host? && (retry_count > 0)
+          use_fallback = can_fallback_to_alternate_ably_host? && retry_count.positive?
 
           conn = if preferred_fallback_connection_for_first_request
-            case retry_count
-            when 0
-              preferred_fallback_connection_for_first_request
-            when 1
-              # Ensure the root host is used first if the preferred fallback fails, see #RSC15f
-              connection(use_fallback: false)
-            end
-          end || connection(use_fallback: use_fallback) # default to normal connection selection process if not preferred connection set
+                   case retry_count
```

Comment:
I'm strongly against this rule. It's extremely inconvenient. It's also inconsistent with other changes like removal of assignments alignment
```
foo      = :foo 
foobar = :foo
```

Reply:
@lukaszsliwa can we please discuss this one

Reply:
@TheSmartnik What would you like to discuss?